howl/model/workspace.py

```diff
@@ -57,12 +44,16 @@ def load_model(self, model: nn.Module, best=True):
         model.load_state_dict(torch.load(
             self.model_path(best=best), lambda s, l: s))
 
-    def write_settings(self, settings):
+    def write_settings(self, settings: HowlSettings = SETTINGS):
+        """Write settings object into JSON file"""
         with (self.path / 'settings.json').open('w') as f:
-            json.dump(gather_dict(settings), f, indent=2)
+            keys_to_ignore = ['_dataset', '_raw_dataset']
```

Comment:
I don't write DatasetSettings or RawDatasetSettings since they only contain the paths to datasets for training, which doesn't seem necessary for reproducibility (user should provide this themselves?)

Reply:
yea this is fine. if I need to ad back in, I will do it